homeassistant/components/overkiz/const.py

```diff
@@ -91,3 +98,11 @@
     OverkizCommandParam.SFC: "sfc",
     OverkizCommandParam.UPS: "ups",
 }
+
+OVERKIZ_UNIT_TO_HA: dict[str, str] = {
+    "core:ElectricalEnergyInKWh": ENERGY_KILO_WATT_HOUR,
```

Comment:
Let's see if we can add these state attributes to enums in pyoverkiz.

Reply:
Done. And there was a lot to do :)